codalab/lib/download_manager.py

```diff
@@ -378,21 +376,19 @@ def _send_netcat_message(self, worker, response_socket_id, uuid, port, message):
             'port': port,
             'message': message,
         }
-        if not self._worker_model.send_json_message(
-            worker['socket_id'], worker['worker_id'], message, 60
-        ):  # dead workers are a fact of life now
+        if not self._worker_model.send_json(message, worker['worker_id']):
             logging.info('Unable to reach worker')
 
     def _get_read_response_stream(self, response_socket_id):
         with closing(self._worker_model.start_listening(response_socket_id)) as sock:
-            header_message = self._worker_model.get_json_message(sock, 60)
+            header_message = self._worker_model.recv_json_message_with_sock(sock, 60)
```

Comment:
rename to `get_json_message`?

Reply:
I do want to keep the `with_sock` portion (I changed it to `with_unix_socket` in this newest version) since it helps make clear that it's using `AF_Unix` sockets. However, I could change `recv` back to `get` -- I thought `recv` was clearer since that's typically the API for getting data from sockets.

Let me know. I can definitely change this.